plugins/source.go

```diff
@@ -46,6 +46,10 @@ const (
 // Add internal columns
 func addInternalColumns(tables []*schema.Table) {
 	for _, table := range tables {
+		c := table.Column("_cq_id")
+		if c != nil {
```

Comment:
nit: since `c` is only used in this block, I would use the `if c := ...; c != nil {` syntax.